src/components/router/router.js

```diff
@@ -5,6 +5,16 @@ import { useSetRouteDetail } from './routerContext';
 import { routerHelpers } from './routerHelpers';
 import { Loader } from '../loader/loader';
 
+/**
+ * Route component loader.
+ *
+ * @memberof Components
+ * @module Router
+ * @property {module} Redirect
```

Comment:
required: this module no longer exists with the recent routing updates, remove it